tests/support/constants.py

```diff
@@ -3,5 +3,38 @@
 TEST_BASE_URL = "https://example.com"
 """Base URL used for the test `httpx.AsyncClient`."""
 
-TEST_GITHUB_WEBHOOK_SECRET = "some-webhook-secret"
-"""Webhook secret used for hashing test github webhook payloads."""
+TEST_GITHUB_REFRESH_APP_SECRET = "some-webhook-secret"
+"""Webhook secret used for hashing test github refresh app webhook payloads."""
+
+TEST_GITHUB_CI_APP_SECRET = "some-ci-webhook-secret"
+"""Webhook secret used for hashing test github ci app webhook payloads."""
+
+TEST_GITHUB_CI_APP_PRIVATE_KEY = """-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I'm sort of surprised that GH hasn't freaked out with "OMG YOU COMMITTED A PRIVATE KEY".

Reply:
Yeah, that's a good point, I wonder why it's not.  I stole this straight out of [times square](https://github.com/lsst-sqre/times-square/blob/main/tests/support/github.py#L15-L43).

Reply:
Turns out secret scanning wasn't enabled in the [security settings](https://github.com/lsst-sqre/mobu/security).  I enabled it, maybe... someone... will see a notification... somewhere...